src/helpers/getPetitionsPageTitle.js

```diff
@@ -0,0 +1,9 @@
+import settings from 'settings';
+
+export default (currentCity) => {
+  const city = currentCity && currentCity.name;
+
+  return city
+    ? `${settings.petitionsPage.titleLocalized} ${city}`
+    : settings.petitionsPage.title;
+};
```

Comment:
This piece of code should get a test as well. But since there is a bit of time pressure and it is working for now, we can add them in a later PR I guess. I was working on a couple of helper tests anyway on a separate branch.